scapy/main.py

```diff
@@ -412,11 +449,29 @@ def update_session(fname=None):
     update_ipython_session(scapy_session)
 
 
+@overload
```

Comment:
Can you explain the rationale to overload `init_session()`?  I did not understand it on my own.

Reply:
Thanks!